list.rb

```diff
@@ -0,0 +1,14 @@
+# Including the module
+require_relative 'my_enumerable'
+
```

Comment:
- [ ] Please make sure to specify the proper file name when importing modules as I noticed a bug in line `2`. There's no file by the name of `my_enumerable` it should instead be `enumerable` this way you can access all the methods you implemented in your enumerable class.